nix/haskell.nix

```diff
@@ -126,6 +126,7 @@ let
         # systemd can't be statically linked
         packages.cardano-config.flags.systemd = !pkgs.stdenv.hostPlatform.isMusl;
         packages.cardano-node.flags.systemd = !pkgs.stdenv.hostPlatform.isMusl;
+        packages.smash.flags.systemd = !pkgs.stdenv.hostPlatform.isMusl;
```

Comment:
This is probably absurd and not needed. Can be fixed in the review cycle.

Reply:
I do not think it is absurd and it probably is need.

See https://github.com/input-output-hk/cardano-db-sync/issues/515